util/createUtil.js

```diff
@@ -9,17 +9,27 @@ const createUtil = (codec, deserialize) => {
     /**
      * Deserialize Ethereum block into the internal representation.
      *
-     * @param {Buffer} serialized - Binary representation of a Ethereum block.
+     * @param {Uint8Array|Array<Uint8Array>} serialized - Binary representation of a Ethereum block.
      * @returns {Object}
      */
-    deserialize,
+    deserialize: (serialized) => {
```

Comment:
Is this change because upstream only supports Buffers and they also support arrays of Buffers?

Probably needs a `const { Buffer } = require('buffer')` too, right?

Reply:
Good spot, I've PR'd it in: https://github.com/ipld/js-ipld-ethereum/pull/70